distribution/xfer/download_test.go

```diff
@@ -21,6 +21,8 @@ import (
 
 const maxDownloadConcurrency = 3
 
+const maxDownloadAttempts = 5
```

Comment:
If we use the functional argument (and also set the default in `NewLayerDownloadManager`), we don't need this `const`

Reply:
Implemented